### PR TITLE
XFormException translation and clean up

### PR DESCRIPTION
**Pull request by benrudolph**
@czue

### CODE REVIEW

File: corehq/apps/app_manager/xform.py [hunk 1]
```diff
@@ -1370,18 +1366,18 @@ def make_delegation_stub_case_block():
 
         if case_block is not None:
             if case.exists():
-                raise XFormException("You cannot use the Case Management UI if you already have a case block in your form.")
+                raise XFormException(_("You cannot use the Case Management UI if you already have a case block in your form."))
```
Comment (dimagimon) [inline corehq/apps/app_manager/xform.py]: E501 line too long (127 > 115 characters)

File: corehq/apps/app_manager/xform.py [hunk 2]
```diff
@@ -20,7 +21,7 @@ def parse_xml(string):
     try:
         return ET.fromstring(string, parser=ET.XMLParser(encoding="utf-8", remove_comments=True))
     except ET.ParseError, e:
-        raise XFormException("Error parsing XML" + (": %s" % str(e)))
+        raise XFormException(_("Error parsing XML: %s") % str(e))
```
Comment (czue) [inline corehq/apps/app_manager/xform.py]: not sure if the translations framework will handle this, but i wonder if all of these should be unicode and not strings

Comment (czue) [inline corehq/apps/app_manager/xform.py]: (merging anyways since this change is better either way)

Comment (millerdev) [inline corehq/apps/app_manager/xform.py]: > i wonder if all of these should be unicode and not strings

:+1:

**Pull request merged**